.cirrus.yml

```diff
@@ -34,12 +34,12 @@ task:
       env:
         use_valgrind: yes
         CFLAGS: -O0 -g
-    - container:
-        image: ubuntu:20.04
-      env:
-        use_valgrind: yes
-        CFLAGS: -O0 -g
-        PGVERSION: 9.6
+    # - container:
+    #     image: ubuntu:20.04
+    #   env:
+    #     use_valgrind: yes
+    #     CFLAGS: -O0 -g
+    #     PGVERSION: 9.6
```

Comment:
@petere what do you think, 9.6 is EOL for more than a year now. It seems reasonable to me to stop officially supporting it (and thus running tests for it). Stopping to support it would allow to make the scram tests always enabled, since any supported PG version has scram support.

Reply:
Maybe. Is it a problem?